### DISCUSSION TITLE
热门页面显示问题

### DISCUSSION THREAD

**Discussion by yishuoVC** [ywmoyue/biliuwp-lite#392] [bug]
### Bug 描述

视频数据不显示

### 操作步骤

```plain text
视频数据不显示
```

### 期望结果

视频数据不显示

### 截图

![image](https://github.com/ywmoyue/biliuwp-lite/assets/106026621/20c12d64-b415-4bb2-a238-650f76560d27)

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
我这里没有复现。请给出日志帮助我们定位这个问题。@yishuoVC

**Reply by yishuoVC** [reply]
[20231104.log](https://github.com/ywmoyue/biliuwp-lite/files/13257334/20231104.log)

**Reply by GD-Slime** [reply]
@ywmoyue 这里请求的应该是 https://app.bilibili.com/x/v2/show/popular/index 但是我这里没什么问题 
你能尝试复现吗

**Reply by ywmoyue** [reply]
@GD-Slime 我复现了，我修复下
